packages/showcases/src/pages/e-commerce.tsx

```diff
@@ -0,0 +1,291 @@
+
+  return (
+    <div className="flex flex-row gap-4">
+      <Filter
+        title="Season"
+        current={season}
+        onSelect={setSeason}
+        options={["winter", "spring", "summer", "fall"]}
+      />
+      <Filter
+        title="Price"
+        current={price}
+        onSelect={setPrice}
+        options={["Under $500", "Under $200"]}
+      />
+      <Filter
+        title="Gender"
+        current={gender}
+        onSelect={setGender}
+        options={["boys", "girls", "men", "women"]}
+      />
+      <Filter
+        title="color"
+        current={color}
+        onSelect={setColor}
+        options={["red", "orange", "yellow", "green", "blue", "white", "black"]}
+      />
+      <Filter
+        title="Category"
+        current={category}
+        onSelect={setCategory}
+        options={["apparel", "accessories"]}
+      />
+    </div>
+  );
+};
+
+const Tag = ({ children }: { children: string }) => (
+  <div className="px-2 bg-gray-500 rounded-full text-white">{children}</div>
+);
+
+const ProductResult = ({ result }: { result: SimpleResult }) => {
+  return (
+    <div className="rounded border h-full flex flex-col">
+      <div className="border-b">
+        <MediaPreview src={result.data} mimeType={result.mimeType} />
+      </div>
+      {result.tags && (
+        <div className="flex-1 flex flex-col p-3">
+          <div className="capitalize font-semibold text-lg">
+            {result.tags.display_name}
+          </div>
+          <div className="text-gray-500 flex-1">
+            <TextPreview
+              text={result.tags.description}
+              size="sm"
+              clampLines={3}
+            />
+          </div>
+          <div className="flex flex-row gap-1 capitalize mt-2">
+            <Tag>{result.tags.usage}</Tag>
+            <Tag>{result.tags.season}</Tag>
+          </div>
+
+          <div className="flex flex-row items-center border-t mt-2 pt-2">
+            <div className="text-2xl font-semibold flex-1">
+              ${result.tags.price}
+            </div>
+            <div className="px-4 py-1 bg-blue-600 rounded-full text-white cursor-pointer flex flex-row items-center text-sm">
+              <ShoppingCartIcon className="h-3.5 inline" />
+              <div className="ml-1">Add to Cart</div>
+            </div>
+          </div>
+        </div>
+      )}
+    </div>
+  );
+};
+
+const EmptyMessage = () => (
+  <div className="text-center text-3xl text-gray-500 py-36">
+    Search for something
+  </div>
+);
+
+const Searching = () => (
+  <div className="mx-auto text-3xl py-36 flex flex-row items-center text-primary-500">
+    <Spinner />
+    <span className="animate-pulse">Searching...</span>
+  </div>
+);
+
+type FilterCondition = {
+  attribute: string;
+  operator: "eq" | "ne" | "lt" | "gt" | "le" | "ge";
+  value: string | number;
+};
+
+export default function Home() {
+  const [url, setURL] = useState<BaseURL | undefined>(
+    "http://35.246.253.36/shop-the-look/"
```

Comment:
I don't think we should make the url dynamic for the showcases.

Reply:
I've updated this